osgp/protocol-adapter-iec61850/osgp-protocol-adapter-iec61850/src/main/java/org/opensmartgridplatform/adapter/protocol/iec61850/infra/networking/services/commands/Iec61850RebootCommand.java

```diff
@@ -23,11 +20,19 @@
 import org.opensmartgridplatform.adapter.protocol.iec61850.infra.networking.helper.NodeContainer;
 import org.opensmartgridplatform.adapter.protocol.iec61850.infra.networking.helper.SubDataAttribute;
 import org.opensmartgridplatform.adapter.protocol.iec61850.services.DeviceMessageLoggingService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class Iec61850RebootCommand {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(Iec61850RebootCommand.class);
 
+    private DeviceMessageLoggingService loggingService;
+
+    public Iec61850RebootCommand(final DeviceMessageLoggingService loggingService) {
+        this.loggingService = loggingService;
+    }
+
     public void rebootDevice(final Iec61850Client iec61850Client, final DeviceConnection deviceConnection)
             throws ProtocolAdapterException {
         final Function<Void> function = new Function<Void>() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce this anonymous class number of lines from 27 to at most 20, or make it a named class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1188)